htdocs/main.inc.php

```diff
@@ -2743,38 +2743,41 @@ function left_menu($menu_array_before, $helppagename = '', $notused = '', $menu_
 		if (!empty($conf->global->MAIN_BUGTRACK_ENABLELINK)) {
 			require_once DOL_DOCUMENT_ROOT.'/core/lib/functions2.lib.php';
 
-			$bugbaseurl = 'https://github.com/Dolibarr/dolibarr/issues/new?labels=Bug';
-			$bugbaseurl .= '&title=';
-			$bugbaseurl .= urlencode("Bug: ");
-			$bugbaseurl .= '&body=';
-			$bugbaseurl .= urlencode("# Instructions\n");
-			$bugbaseurl .= urlencode("*This is a template to help you report good issues. You may use [Github Markdown](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/) syntax to format your issue report.*\n");
-			$bugbaseurl .= urlencode("*Please:*\n");
-			$bugbaseurl .= urlencode("- *replace the bracket enclosed texts with meaningful information*\n");
-			$bugbaseurl .= urlencode("- *remove any unused sub-section*\n");
-			$bugbaseurl .= urlencode("\n");
-			$bugbaseurl .= urlencode("\n");
-			$bugbaseurl .= urlencode("# Bug\n");
-			$bugbaseurl .= urlencode("[*Short description*]\n");
-			$bugbaseurl .= urlencode("\n");
-			$bugbaseurl .= urlencode("## Environment\n");
-			$bugbaseurl .= urlencode("- **Version**: ".DOL_VERSION."\n");
-			$bugbaseurl .= urlencode("- **OS**: ".php_uname('s')."\n");
-			$bugbaseurl .= urlencode("- **Web server**: ".$_SERVER["SERVER_SOFTWARE"]."\n");
-			$bugbaseurl .= urlencode("- **PHP**: ".php_sapi_name().' '.phpversion()."\n");
-			$bugbaseurl .= urlencode("- **Database**: ".$db::LABEL.' '.$db->getVersion()."\n");
-			$bugbaseurl .= urlencode("- **URL(s)**: ".$_SERVER["REQUEST_URI"]."\n");
-			$bugbaseurl .= urlencode("\n");
-			$bugbaseurl .= urlencode("## Expected and actual behavior\n");
-			$bugbaseurl .= urlencode("[*Verbose description*]\n");
-			$bugbaseurl .= urlencode("\n");
-			$bugbaseurl .= urlencode("## Steps to reproduce the behavior\n");
-			$bugbaseurl .= urlencode("[*Verbose description*]\n");
-			$bugbaseurl .= urlencode("\n");
-			$bugbaseurl .= urlencode("## [Attached files](https://help.github.com/articles/issue-attachments) (Screenshots, screencasts, dolibarr.log, debugging informations…)\n");
-			$bugbaseurl .= urlencode("[*Files*]\n");
-			$bugbaseurl .= urlencode("\n");
-
+			if (!empty($conf->global->MAIN_BUGTRACK_URL)) {
```

Comment:
Do not add a new setup for this. Reuse MAIN_BUGTRACK_ENABLELINK
MAIN_BUGTRACK_ENABLELINK = 1, use default behaviour to github
MAIN_BUGTRACK_ENABLELINK = http://   use it as url